docs/modules.adoc

```diff
@@ -17,13 +17,10 @@ https://github.com/spockframework/spock/tree/master/spock-tapestry/src/test/groo
 == Unitils Module
 
 Integration with the http://www.unitils.org/[Unitils] library. For examples see the specs in the
-https://github.com/spockframework/spock/tree/master/spock-unitils/src/test/groovy/org/spockframework/unitils/dbunit[codebase].
+https://github.com/spockframework/spock/tree/master/spock-unitils/src/test/groovy/org/spockframework/unitils[codebase].
```

Comment:
what is wrong with `https://github.com/spockframework/spock/tree/master/spock-unitils/src/test/groovy/org/spockframework/unitils/dbunit` ?

Reply:
Hm... Looks like it's fine. I don't remember why I had to change it, but there doesn't seem to be a reason to do so now.